src/engine/ruleset/integrations/windows/rules/defense-evasion-deleting-websvr-access-logs.yml

```diff
@@ -0,0 +1,41 @@
+name: rule/defense-evasion-deleting-websvr-access-logs/0
+
+metadata:
+  module: Windows
+  title: Defense Evasion.
+  description: Identifies the deletion of WebServer access logs. This may indicate an attempt to evade detection or destroy forensic
+    evidence on a system.
+  versions: [Vista, "7", "8", "10", "11", Server 2012, Server 2016, Server 2019, Server 2022]
+  compatibility: This rule has been tested on Wazuh version 4.5.
+  author:
+    name: Wazuh, Inc.
+    date: 2023/09/15
+  references:
+    - "https://attack.mitre.org/tactics/TA0005/"
+
+check : >-
+    array_contains($event.type, deletion) AND array_contains($event.category, file)
+    AND (regex_match($file.path, '^[A-Za-z]:\\\\inetpub\\\\logs\\\\LogFiles\\\\[A-Za-z]+\.log')
+        OR regex_match($file.path, '^/var\/log\/apache[A-Za-z]+\/access\.log')
+        OR $file.path == /etc/httpd/logs/access_log
+        OR $file.path == /var/log/httpd/access_log
+        OR regex_match($file.path, '^/var\/www\/[A-Za-z]+\/logs\/access\.log'))
+
+normalize:
+  - map:
+      - event.risk_score: 47
+
+      - rule.description: Identifies the deletion of WebServer access logs. This may indicate an attempt to evade detection or destroy forensic
+          evidence on a system.
+      - rule.license: Wazuh Inc.
+      - rule.name: WebServer Access Logs Deleted
+
+      - threat.framework: MITRE ATT&CK
+      - threat.tactic.id: array_append(TA0005)
+      - threat.tactic.name: array_append(Defense Evasion)
```

Comment:
Wouldn't it be more legible this way? What do you think?: 
`- threat.tactic.name: array_append("Defense Evasion")`

Reply:
needless